### PR TITLE
hw6 completed

### PR DESCRIPTION
**Pull request by strrust**

### CODE REVIEW

File: src/main/java/ru/training/at/hw6/forms/JdiMetalsColorsForm.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    public RadioButtons oddNumbers;
+
+    @UI("[name=custom_radio_even]")
+    public RadioButtons evenNumbers;
+
+    @UI("#submit-button")
+    public Button submitButton;
+
+    public void fillInMetalsColorsForm(MetalsColorsTestData data) {
+        int i;
```
Comment (shoggy) [inline src/main/java/ru/training/at/hw6/forms/JdiMetalsColorsForm.java:47]: This variable is not used outside of loops, I think there is no need to put it to such wide scope

File: src/main/java/ru/training/at/hw6/forms/JdiMetalsColorsForm.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    @UI("[name=custom_radio_odd]")
+    public RadioButtons oddNumbers;
+
+    @UI("[name=custom_radio_even]")
+    public RadioButtons evenNumbers;
+
+    @UI("#submit-button")
+    public Button submitButton;
+
+    public void fillInMetalsColorsForm(MetalsColorsTestData data) {
```
Comment (shoggy) [inline src/main/java/ru/training/at/hw6/forms/JdiMetalsColorsForm.java:46]: `Form` class has a special method for this type of action, override it, please

File: src/main/java/ru/training/at/hw6/pages/JdiHomePage.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    public Menu headerMenu;
+
+    public void loginToSite(User user) {
+        userIcon.click();
+        jdiLoginForm.login(user);
+    }
+
+    public void checkLoggedUserName(User user) {
+        userName.is().text(user.userName);
+    }
```
Comment (shoggy) [inline src/main/java/ru/training/at/hw6/pages/JdiHomePage.java:38]: page objects layer should not contain assertions

\+ same for another classes

**Review by shoggy — CHANGES_REQUESTED** [review]

File: src/main/java/ru/training/at/hw6/forms/JdiMetalsColorsForm.java (hunk 1 above)
Comment (strrust) [reply src/main/java/ru/training/at/hw6/forms/JdiMetalsColorsForm.java]: Replaced var init to cycles scope only

File: src/main/java/ru/training/at/hw6/forms/JdiMetalsColorsForm.java (hunk 2 above)
Comment (strrust) [reply src/main/java/ru/training/at/hw6/forms/JdiMetalsColorsForm.java]: overrided like a fill method

File: src/main/java/ru/training/at/hw6/pages/JdiHomePage.java (hunk 3 above)
Comment (strrust) [reply src/main/java/ru/training/at/hw6/pages/JdiHomePage.java]: assertions was moved to test directly

**Review by shoggy — APPROVED** [review]

**Pull request merged**